contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -1609,3 +1609,27 @@ Example: "must be greater than \`request\`".
 be less than 256", "must be greater than or equal to 0".  Do not use words
 like "larger than", "bigger than", "more than", "higher than", etc.
 * When specifying numeric ranges, use inclusive ranges when possible.
+
+
+## Automatic Resource Allocation And Deallocation
+
+API objects often are union object containing the following:
```

Comment:
Should this be part of or maybe cross-ref the existing section on unions?  ca. L572